--- ./scripts/bootstrap/compile.sh.oldfile      2019-05-09 06:19:43.508791854 -0400
+++ ./scripts/bootstrap/compile.sh.newfile      2019-05-09 06:23:32.449966357 -0400
@@ -127,7 +127,7 @@
   # Useful if your system chooses too small of a max heap for javac.
   # We intentionally rely on shell word splitting to allow multiple
   # additional arguments to be passed to javac.
-  run "${JAVAC}" -classpath "${classpath}" -sourcepath "${sourcepath}" \
+  run "${JAVAC}" -J-Xms1g -J-Xmx1g -classpath "${classpath}" -sourcepath "${sourcepath}" \
       -d "${output}/classes" -source "$JAVA_VERSION" -target "$JAVA_VERSION" \
       -encoding UTF-8 ${BAZEL_JAVAC_OPTS} "@${paramfile}"       
