--- Makefile.oldfile    2019-06-24 10:33:38.778994198 +0000
+++ Makefile.newfile    2019-06-26 12:54:12.248643919 +0000
@@ -35,7 +35,7 @@

 # Removed 'bazel shutdown' as it could cause CircleCI to hang
 build:
-       PATH=$(PATH) CC=$(CC) CXX=$(CXX) bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) $(BAZEL_TARGETS)
+       PATH=$(PATH) CC=$(CC) CXX=$(CXX) bazel $(BAZEL_STARTUP_ARGS) build -c opt $(BAZEL_BUILD_ARGS) $(BAZEL_TARGETS) --host_javabase=@local_jdk//:jdk

 # Build only envoy - fast
 build_envoy:
