--- ./Makefile.oldfile  2019-05-09 06:45:04.826038421 -0400
+++ ./Makefile.newfile  2019-05-09 06:47:27.185428866 -0400
@@ -34,7 +34,7 @@
 PATH := /usr/lib/llvm-7/bin:$(PATH)

 build:
-       PATH=$(PATH) CC=$(CC) CXX=$(CXX) bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) $(BAZEL_TARGETS)
+       PATH=$(PATH) CC=$(CC) CXX=$(CXX) bazel $(BAZEL_STARTUP_ARGS) build -c opt $(BAZEL_BUILD_ARGS) $(BAZEL_TARGETS) --host_javabase=@local_jdk//:jdk
        @bazel shutdown

 # Build only envoy - fast
